cmd/main.go

```diff
@@ -195,6 +196,12 @@ func main() {
 			log.Error(err, "failed to initialize node commonService", "controller", "Node")
 			os.Exit(1)
 		}
+		staticRouteService, err := staticroute.InitializeStaticRoute(commonService)
+		if err != nil {
+			log.Error(err, "failed to initialize staticroute commonService", "controller", "StaticRoute")
+			os.Exit(1)
+		}
+		staticRouteService.VPCService = vpcService
```

Comment:
If someone want to use staticRouteService, this means that he has to remember to assign vpcService though staticRouteService had be initialized by `staticroute.InitializeStaticRoute`. This doesn't make sense.